src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -907,6 +907,239 @@ def test_interface_alignment(self):
                 self.check_nwbfile_temporal_alignment()
 
 
+class MedPCInterfaceMixin(DataInterfaceTestMixin, TemporalAlignmentMixin):
```

Comment:
@CodyCBakerPhD @pauladkisson 

Hey guys I am working on testing and looking at this.

Looks like mose of these methods were copied directly from DataInterfaceTestMixing and they were not necessary.  Looking at `check_configure_backend_for_equivalent_nwbfiles` for example, the metadata line was removed but the original function should handle that fine.

Was this re-defined  for other purpose that I am missing?

Reply:
Yes, these tests were redefined to work on metadata passed in from the user (which is critical for MedPCInterface) rather than just interface.get_metadata()